solutions/tier3/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/nat.yaml

```diff
@@ -64,9 +61,6 @@ spec:
   routerRef:
     name: host-project-id-nane2-router # kpt-set: ${host-project-id}-nane2-router
   sourceSubnetworkIpRangesToNat: ALL_SUBNETWORKS_ALL_IP_RANGES
-  logConfig:
-    enable: true
-    filter: ALL
```

Comment:
Is there a reason this is being removed? We should not be altering these packages as approvals base on the current code as currently in flight.